### DISCUSSION TITLE
如何获取公众号ID 页面显示点不开  下载的macbook OS版本

### DISCUSSION THREAD

**Discussion by CAROLINE12312312** [qiye45/wechatDownload#264] [General]
<img width="970" alt="Screenshot 2024-12-20 at 12 31 31" src="https://github.com/user-attachments/assets/553a4a2a-d487-4bfe-beaf-f0a938b2fc07" />
![Uploading Screenshot 2024-12-20 at 12.35.25.png…]()

**Comment by CAROLINE12312312** [comment]
<img width="255" alt="Screenshot 2024-12-20 at 12 35 25" src="https://github.com/user-attachments/assets/62888d53-0161-4f09-84d4-1c588e507ba1" />
dian'n电脑版本

**Reply by CAROLINE12312312** [reply]
第一次在github上提问 格式还不太熟悉 感谢开发大大

**Reply by qiye45** [reply]
https://github.com/qiye45/wechatDownload/issues/147
